squad/core/management/commands/compute_project_statuses.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import logging
+
+from datetime import timedelta, datetime
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from squad.core.models import Project, Build, ProjectStatus
+
+
+logger = logging.getLogger()
+environments_cache = {}
+
+
+def valid_date(date):
+    try:
+        return datetime.strptime(date, "%Y-%m-%d")
+    except ValueError:
+        msg = "'{0}' is not a valid date.".format(
+            date)
+        raise argparse.ArgumentTypeError(msg)
+
+
+class Command(BaseCommand):
+
+    help = """Compute metric summaries per build per environment"""
```

Comment:
same help msg as computer_build_summaries.py ?

Reply:
copy/paste error. Thanks for spotting.